### DISCUSSION TITLE
Add alternate format for inline events

### DISCUSSION THREAD

**Discussion by dxcore35** [davish/obsidian-full-calendar#375] [Feature Requests]
Please change formatting and logic of [startTime:: XX:XX] [endTime:: XX:XX]

For example use standard task format HH-mm and variable defining length by cleaning of code 12:05 duration: 1.5h
`- [ ]  12:05, 1.5h`

The main reason is that it require to much typing when entering task in note. 

---
You should please utilize the current plugins naming convention, especialy from the https://obsidian-tasks-group.github.io/obsidian-tasks/ and also make the writing of the task as less burden as possible. The colaboration with Obsidian task developer will be very fruitfull, as he already created GUI for editing task properties. He is just missing the lenght parameter!

This is just sugestions. I know you want to utilize the YAML inline fields, but except your plugin I don't think other people will be querieng this for task. As they have Tasker-plugin with huge options. Even for long term I advice to contact the developer and find common ground.

**Comment by davish** [comment]
Replicating my response on the other issue: 
>  I won't be changing the time format from startTime/endTime at this point. While startTime/duration would also work, the goal is for inline event metadata to be as close to the metadata for frontmatter events as possible.

You can follow along about integrating with the Tasks plugin in issue #219.

While the `startTime`/`endTime` format is here to stay, that doesn't mean that we can't add alternate formatting. I will keep this issue open as an enhancement.

**Comment by sandorex** [comment]
> that doesn't mean that we can't add alternate formatting. I will keep this issue open as an enhancement.

I'd recommend [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), while a bit weird looking it is already standardized and parsable from most libraries
